tests/apps/epi/test_epi_api.py

```diff
@@ -10,6 +10,25 @@
 
 @pytest.mark.django_db
 class TestEpiAssessmentViewset:
+    def _test_heatmap(self, rewrite_data_files: bool, slug: str, key: int, unpublished: bool):
```

Comment:
this code is almost identical to `test_full_export` - can all 3 tests be consolidated like  `_test_animal_export` in `test_animal_api.py.`